api/schema.py

```diff
@@ -0,0 +1,29 @@
+from drf_spectacular.openapi import AutoSchema
+from rest_framework.permissions import SAFE_METHODS
+
+from api.serializers.errors import ValidationErrorSerializer
+
+
+class CustomSchema(AutoSchema):
+    """
```

Comment:
this is something I learnt recently: a multiline docstring [should contain a summary line](https://peps.python.org/pep-0257/#multi-line-docstrings) that fits on one line, followed by a blank line, followed by the more detailed explanation.

Reply:
I agree here